docs/events_handling.md

```diff
@@ -0,0 +1,101 @@
+
+# Events handling
+
+## Subscribed events
+
+Precaution needs to subscribe to a number of events
+to satisfy different use cases.
+
+The events are separated into two groups `PullRequestEvent` and `CheckRunEvent/CheckSuiteEvent`.
+
+### PullRequestEvent
+
+Official GitHub documentation: [PullRequest event](https://developer.github.com/v3/activity/events/types/#pullrequestevent)
```

Comment:
Is it "PullRequest event" or "PullRequestEvent"?

Reply:
Yep, my mistake.